app/demo-loop/LDemoRun.cc

```diff
@@ -0,0 +1,127 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2021 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file LDemoRun.cc
+//---------------------------------------------------------------------------//
+#include "LDemoRun.hh"
```

Comment:
I probably missed it but I did not run into the expansion of "L" in the doc ... humm ah right looking at the path name "demo-loop" it probably means "Loop" but then we have "demo-loop/LoopDemo..." .. not important but odd :)

Reply:
Yeah sorry, I modeled it after the `KNDemo` prefix inside the `demo-interactor`... so consistent with that but not really sensible 😔